<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ChangeProperties</class>
  <widget class="QDialog" name="ChangeProperties" >
    <property name="objectName" >
      <string notr="true" >ChangeProperties</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>728</width>
        <height>584</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Change Control Properties</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QTabWidget" name="tabWidget" >
          <property name="objectName" >
            <string notr="true" >tabWidget</string>
          </property>
          <widget class="QWidget" name="tab" >
            <property name="objectName" >
              <string notr="true" >tab</string>
            </property>
            <attribute name="title" >
              <string>&amp;Properties</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <property name="objectName" >
                <string notr="true" >unnamed</string>
              </property>
              <property name="margin" >
                <number>11</number>
              </property>
              <property name="spacing" >
                <number>6</number>
              </property>
              <item>
                <widget class="QTreeWidget" name="listProperties" >
                  <property name="objectName" >
                    <string notr="true" >listProperties</string>
                  </property>
                  <column>
                    <property name="text" >
                      <string>Property</string>
                    </property>
                    <property name="clickable" >
                      <bool>false</bool>
                    </property>
                    <property name="resizeable" >
                      <bool>true</bool>
                    </property>
                  </column>
                  <column>
                    <property name="text" >
                      <string>Type</string>
                    </property>
                    <property name="clickable" >
                      <bool>false</bool>
                    </property>
                    <property name="resizeable" >
                      <bool>true</bool>
                    </property>
                  </column>
                  <column>
                    <property name="text" >
                      <string>Value</string>
                    </property>
                    <property name="clickable" >
                      <bool>false</bool>
                    </property>
                    <property name="resizeable" >
                      <bool>true</bool>
                    </property>
                  </column>
                </widget>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="objectName" >
                    <string notr="true" >unnamed</string>
                  </property>
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <property name="spacing" >
                    <number>6</number>
                  </property>
                  <item>
                    <widget class="QLabel" name="valueLabel" >
                      <property name="objectName" >
                        <string notr="true" >valueLabel</string>
                      </property>
                      <property name="text" >
                        <string>Property &amp;Value:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>editValue</cstring>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="editValue" >
                      <property name="objectName" >
                        <string notr="true" >editValue</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QToolButton" name="buttonSet" >
                      <property name="objectName" >
                        <string notr="true" >buttonSet</string>
                      </property>
                      <property name="text" >
                        <string>&amp;Set</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="tab" >
            <property name="objectName" >
              <string notr="true" >tab</string>
            </property>
            <attribute name="title" >
              <string>Property Edit &amp;Requests</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <property name="objectName" >
                <string notr="true" >unnamed</string>
              </property>
              <property name="margin" >
                <number>11</number>
              </property>
              <property name="spacing" >
                <number>6</number>
              </property>
              <item>
                <widget class="QTreeWidget" name="listEditRequests" >
                  <property name="objectName" >
                    <string notr="true" >listEditRequests</string>
                  </property>
                  <column>
                    <property name="text" >
                      <string>Property</string>
                    </property>
                    <property name="clickable" >
                      <bool>false</bool>
                    </property>
                    <property name="resizeable" >
                      <bool>false</bool>
                    </property>
                  </column>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer7" >
              <property name="sizeHint" >
                <size>
                  <width>1</width>
                  <height>1729677848</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="PushButton7" >
              <property name="objectName" >
                <string notr="true" >PushButton7</string>
              </property>
              <property name="text" >
                <string>C&amp;lose</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
